guides/schematics.md

```diff
@@ -6,22 +6,13 @@ Schematics come packaged with Angular Material, so once you have
 installed the npm package, they will be available via the Angular CLI.
 
 If you run it will automatically install Angular Material for you
-and run the shell schematic.
+and run the install schematic.
```

Comment:
It is actually called "shell" in the `collection.json`. We should change that, too.